docs/userguide/labels-custom-metadata.md

```diff
@@ -59,15 +59,14 @@ notation. Use the following guidelines to name your keys:
   cumbersome namespaces on the command-line.
 
 
-These are guidelines and Docker does not *enforce* them. Failing following these
-guidelines can result in conflicting labels. If you're building a tool that uses
-labels, you *should* use namespaces for your label keys.
+These are simply guidelines and Docker does not *enforce* them. However, for the
+the benefit of the community, you *should* use namespaces for your label keys.
```

Comment:
There's one "the" too many here (one at the start of this line, one at the end of the previous line)